bin/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#
+# Setup Python virtual environment
+# and install dependencies.
+#
+virtualenv venv
+source venv/bin/activate
+pip install --upgrade pip
+pip install -r requirements.txt
+easy_install --upgrade requests
+pip install requests[security]
```

Comment:
* Creates a Python virtual environment.
* Makes sure the latest version of `pip` is installed.
* Installs requirements.
* Upgrades `requests` via easy_install to get around this [issue](http://stackoverflow.com/questions/28987891/patch-pyopenssl-for-sslv3-issue).
* Install requests security add ons to prevent security warnings to be issued when connecting to HDX's HTTPS.